### DISCUSSION TITLE
Isn't file content type and file encoding different?

### DISCUSSION THREAD

**Discussion by SmartManoj** [chatwoot/chatwoot#1468] [General]
https://github.com/chatwoot/chatwoot/blob/a5ceddc454efe56a0837053bd8ad1e6b970e4270/app/services/twilio/incoming_message_service.rb#L106

**Comment by pranavrajs** [comment]
@SmartManoj I guess we are parsing the encoding in the service. I don't think this will be a problem. Can you please elaborate on what the issue is?
